src/index.js

```diff
@@ -1,10 +1,14 @@
 // @flow
 import React from "react";
 
+declare function __webpack_require__(pathOrId: string | number): any;
 type GenericComponent<Props> = Class<React.Component<{}, Props, mixed>>;
 type LoadedComponent<Props> = GenericComponent<Props>;
 type LoadingComponent = GenericComponent<{}>;
 
+let isWebpack = typeof __webpack_require__ !== "undefined";
+let requireFn = isWebpack ? __webpack_require__ : module.require;
```

Comment:
Thanks for checking, but I've verified that this works fine in node:

```javascript
// temp.js
const requireFn = module.require;

console.log(requireFn('fs'));
console.log(requireFn('./package.json'));
console.log(requireFn('./lib/index'));
```

Are there other contexts where this could be a problem though?